src/app/project/controllers/collect-records.ctrl.js

```diff
@@ -211,11 +211,17 @@ angular.module('app.project').controller('CollectRecordsCtrl', [
           display: 'Sample Date',
           sortable: true,
           formatter: function(v) {
-            let val = '';
-            if (v) {
-              val = $filter('date')(new Date(v), 'dd-MMM-yyyy');
+            let dateResult = '';
+            const dateVal = v && v.split('-').map(val => Number(val));
+            if (dateVal && dateVal.length >= 3) {
+              const newDateVal = new Date(
+                dateVal[0],
+                dateVal[1] - 1,
```

Comment:
Why `- 1`? Is the date not correct coming back from the API?

Reply:
I'm changing to move away from date-string approach to passing number arguments to `new Date()` where 2nd argument is month (which index starts at 0 for January)